docs/mindsdb_sql/functions/from_env.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: The from_env() Function
+sidebarTitle: The from_env() Function
+---
+
+MindsDB provides the `from_env()` function that lets users pull values from the environment variables into MindsDB.
+
+## Usage
+
+Here is how to use the `from_env()` function.
+
+* Save the environment variable into a variable using the `from_env()` function (or saving the value directly).
+
+```sql
+SET @my_env_var  = from_env("MY_ENV_VAR")
```

Comment:
due to security conserns (possible access to artitrary env variables), only variables with 
MDB_ prefix  is possible to use

Reply:
thanks @ea-rus 

updated